src/unix/kqueue.c

```diff
@@ -230,10 +239,13 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
 
       revents = 0;
 
-      if (ev->filter == EVFILT_READ) {
+      if (ev->filter == EVFILT_READ || ev->filter == EV_OOBAND) {
         if (w->pevents & POLLIN) {
           revents |= POLLIN;
           w->rcount = ev->data;
+        } else if(w->pevents & UV__POLLPRI) {
+          revents |= UV__POLLPRI;
+          w->rcount = ev->data;
```

Comment:
Same comment here. Maybe having a helper function for `EVFILT_READ`, `EVFILT_WRITE` and `EV_OOBAND` could be a good idea, as they are basically doing the same thing?